functions/__sf_section_kubecontext.fish

```diff
@@ -11,25 +11,33 @@ function __sf_section_kubecontext -d "Display the kubernetes context"
 	# ------------------------------------------------------------------------------
 
 	__sf_util_set_default SPACEFISH_KUBECONTEXT_SHOW true
+	__sf_util_set_default SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW true
 	__sf_util_set_default SPACEFISH_KUBECONTEXT_PREFIX "at "
 	__sf_util_set_default SPACEFISH_KUBECONTEXT_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
 	# Additional space is added because ☸️ is wider than other symbols
 	# See: https://github.com/denysdovhan/spaceship-prompt/pull/432
 	__sf_util_set_default SPACEFISH_KUBECONTEXT_SYMBOL "☸️  "
 	__sf_util_set_default SPACEFISH_KUBECONTEXT_COLOR cyan
 
+
 	# ------------------------------------------------------------------------------
 	# Section
 	# ------------------------------------------------------------------------------
 
 	# Show current kubecontext
 	[ $SPACEFISH_KUBECONTEXT_SHOW = false ]; and return
-
 	# Ensure the kubectl command is available
 	type -q kubectl; or return
 
 	set -l kube_context (kubectl config current-context 2>/dev/null)
 	[ -z $kube_context ]; and return
+	
+	if test "$SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW" = "true"
```

Comment:
I think we may want to have this test for both `"$SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW" = "true"` _and_ `$kube_context != "default"` so that we can avoid running `kubectl config view` unnecesarily.

Do you think this would work?

```suggestion
	if test "$SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW" = "true" -a "$kube_context" != "default"
```

Reply:
I am not that deep into `fish`. Does this work as an `and`?
Did as suggested, unit test are going through 👍